index.html

```diff
@@ -0,0 +1,555 @@
+                            <div class="fund">
+                                <p> $1.5M </p>
+                                <small>SEED FUND</small>
+                            </div>
+                            <div class="fund2">
+                                <p><i class="tag icon"></i> Real Time, +3</p>
+                                <p><i class="tag icon"></i> BIP Capital </p>
+                            </div>
+                            <div class="fund3">
+                                <small> Alabama based client relationship management Company case status raises $1.5M on PULSE 2.0 </small>
+                            </div>
+                        </div>
+                        <div class="fund-item">
+                            <img src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1120186370171920384%2FV2h7uPGj.png&signature=8a477026b39431387731f7417b289b3d" alt="Ride-logo">
+                            <h4>Ride Report</h4>
+                            <div class="fund">
+                                <p> $10M </p>
+                                <small>SERIES A</small>
+                            </div>
+                            <div class="fund2">
+                                <p><i class="tag icon"></i> Transportation</p>
+                                <p><i class="tag icon"></i> Homebrew, +3 </p>
+                            </div>
+                            <div class="fund3">
+                                <small> Ride report raises $10M in Series A funding on FinSMEs </small>
+                            </div>
+                        </div>
+                        <div class="fund-item">
+                            <img src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F918584500703121409%2FO0K-YncL.jpg&signature=57ff0a21add57d87f92c75afb4741ac6" alt="Harbour-logo">
+                            <h4>Harbour Biomed</h4>
+                            <div class="fund">
+                                <p> $67.4M </p>
+                                <small>SERIES B</small>
+                            </div>
+                            <div class="fund2">
+                                <p><i class="tag icon"></i> Biotechnology</p>
+                                <p><i class="tag icon"></i> Legend Capital, +8 </p>
+                            </div>
+                            <div class="fund3">
+                                <small> Harbour Biomed raises $67.4M to treat Corona Virus and Cancer on Silicon Canals - European technology News </small>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="tech-news">
+                    <div class="apps-info">
+                        <h1>Apps <i class="chevron right icon"></i></h1>
+                        <div class="app-content">
```

Comment:
- [ ] All the articles in this section should have a larger image when in mobile screen size. Please adjust it.
![Screenshot from 2020-03-18 12-12-42](https://user-images.githubusercontent.com/51926937/76993089-ed9be980-6911-11ea-8041-da1098df49b8.png)

Original:
![Screenshot from 2020-03-18 12-12-35](https://user-images.githubusercontent.com/51926937/76993100-effe4380-6911-11ea-8174-3528484568a1.png)


- [ ] Check the original width and padding of the elements in this section, yours look less wide and they should occupy all the width of the screen.